pkg/uprobetracer/tracer.go

```diff
@@ -57,14 +57,12 @@ const (
 	ProgUretprobe
 )
 
-type inodeUUID struct {
-	token uint64
-}
-
-func getInodeUUID(file *os.File) (inodeUUID, error) {
-	// TODO: use `kfilefields` to get a unique token for each inode,
-	// here we just use `fd` as token, making each file has a distinct token.
-	return inodeUUID{uint64(file.Fd())}, nil
+func getInodeUUID(file *os.File) (uint64, error) {
+	token, err := kfilefields.ReadRealInodeFromFd(int(file.Fd()))
+	if err != nil {
+		return 0, err
+	}
+	return token, nil
```

Comment:
Since this function is just a wrapper around `ReadRealInodeFromFd`, it could be removed and the caller can call `ReadRealInodeFromFd` directly.